core/src/epicli/data/common/ansible/playbooks/roles/applications/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
+---
 postgresql_database_port: "5432"
-configuration_temp_directory: "/home/{{ admin_user.name }}/epiphany-auth"
-default_namespace: "default"
\ No newline at end of file
+default_namespace: "default"
+deployment_tmp_root_dir: "{{ vault_location }}/../apps"  # "inventory_dir" cannot be used here
```

Comment:
could you add a bit more information why?

Reply:
I added description in the commit. If that's not what you meant then I can add it here as well. :)